jina/peapods/remote.py

```diff
@@ -47,17 +47,24 @@ def spawn_remote(self, host: str, port: int, **kwargs) -> Optional[str]:
 
     def loop_body(self):
         if self.remote_id:
-            self.logger.success(f'created remote pea with id {colored(self.remote_id, "cyan")}')
+            self.logger.success(f'created remote {self.api.kind} with id {colored(self.remote_id, "cyan")}')
             self.set_ready()
             self.api.log(self.remote_id)
         else:
             self.logger.error(f'fail to create {self.__class__.__name__} remotely')
             self.is_shutdown.set()
 
-    def loop_teardown(self):
+    def delete_remote(self):
         if hasattr(self, 'api') and self.api.is_alive and self.remote_id:
             self.api.delete(self.remote_id)
 
+    def loop_teardown(self):
+        self.delete_remote()
+
+    def close(self):
+        # close is `pass` here to avoid sending a terminate signal to the remote via zmq
```

Comment:
Why should we avoid sending this signal?

Reply:
not sure, @deepankarm

Reply:
The remote Pea/Pod context is closed using `api.delete()`, which calls `.close()` on remote.  We're not relying on zmq to send the terminate signal

Reply:
why the `self.delete_remote` is not called in `close`?